packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1399,6 +1399,20 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   }
 
+  /**
+   * Post metrics for events, provide the ability to send Call Analyzer for the request of third-party.
+   * @public
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  // eslint-disable-next-line valid-jsdoc
```

Comment:
why do we need this? I think the "@param" string is missing for eventName

Reply:
added.